### ISSUE TITLE
Use new 'ok:<rule-id>' true negative annotation in tests

### ISSUE THREAD

**Issue by mschwager** [returntocorp/sgrep-rules#822]
True negative annotations were recently added to Semgrep: https://github.com/returntocorp/semgrep/pull/1591.

We should update our rule tests to utilize this new annotation. This will help prevent false positives. The testing process will error out when lines are reported with a true negative annotation. The problem is, we currently have a lot of ok'ed lines in our tests:

```
$ rg ' ok' | wc -l
564
```

It may be possible to automate this change with `sed` and some parsing of the filename or existing `ruleid:<rule-id>` annotations.

**Comment by mschwager** [comment]
Here's an example using the new annotation: https://github.com/returntocorp/semgrep-rules/pull/812

**Comment by mschwager** [comment]
We did the thing: https://github.com/returntocorp/semgrep-rules/pull/944 and https://github.com/returntocorp/semgrep-rules/pull/933
